Assets/Scripts/Authoring/PlayerMovementAuthoringComponent.cs

```diff
@@ -89,11 +112,15 @@ public void Convert(Entity entity, EntityManager dstManager, GameObjectConversio
             dstManager.AddComponentData(entity, new KCCJumping()
             {
                 jumpForce = this.jumpForce,
+                jumpGraceTime = this.jumpGraceTime,
+                jumpCooldown = this.jumpCooldown,
+                timeElapsedSinceJump = this.timeElapsedSinceJump,
```

Comment:
Just set this to zero by default if it's not causing errors.

Reply:
Yep, will set that to zero in a moment!

Reply:
Time elapsed removed since it is set by the script.